tests/platform_tests/test_reboot.py

```diff
@@ -56,7 +56,7 @@ def teardown_module(duthosts, enum_rand_one_per_hwsku_hostname, conn_graph_facts
     check_interfaces_and_services(duthost, interfaces, xcvr_skip_list)
 
 
-def reboot_and_check(localhost, dut, interfaces, xcvr_skip_list,
+def reboot_and_check(localhost, dut, conn_graph_facts, xcvr_skip_list, duthosts,
```

Comment:
changing arguments will cause issue -- please also update the following files that has call to reboot_and_check

platform_tests/test_cont_warm_reboot.py
platform_tests/test_power_off_reboot.py